### PR TITLE
Modified Docs as required in issue #223

### PR DESCRIPTION
**Pull request by YashJipkate**

### CODE REVIEW

**Comment by EdgeKing810** [comment]
Why did you leave so many blank lines? :/
And some lines have a looooot of identation at the start...

File: docs/build/html/general/freeradius.html [hunk 1]
```diff
@@ -0,0 +1,727 @@
...
+
+    <script type="text/javascript" id="documentation_options" data-url_root="../" src="../_static/documentation_options.js"></script>
+    <script type="text/javascript" src="../_static/jquery.js"></script>
+    <script type="text/javascript" src="../_static/underscore.js"></script>
+    <script type="text/javascript" src="../_static/doctools.js"></script>
+
+
+
+
+    <script type="text/
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: No empty line at the end of a file.

File: docs/build/html/general/freeradius.html [hunk 2]
```diff
@@ -0,0 +1,727 @@
...
+
+
+
+
+
+    <script type="text/javascript" id="documentation_options" data-url_root="../" src="../_static/documentation_options.js"></script>
+    <script type="text/javascript" src="../_static/jquery.js"></script>
+    <script type="text/javascript" src="../_static/underscore.js"></script>
+    <script type="text/javascript" src="../_static/doctools.js"></script>
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

File: docs/build/html/general/freeradius.html [hunk 3]
```diff
@@ -0,0 +1,727 @@
...
+                    </footer>
+
+                </div>
+            </div>
+
+        </section>
+
+    </div>
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

File: docs/build/html/general/freeradius.html [hunk 4]
```diff
@@ -0,0 +1,727 @@
...
+                <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
+                <a href="../index.html">django-freeradius</a>
+
+            </nav>
+
+
+            <div class="wy-nav-content">
+
+                <div class="rst-content">
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

File: docs/build/html/general/freeradius.html [hunk 5]
```diff
@@ -0,0 +1,727 @@
...
+        <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">
+
+
+            <nav class="wy-nav-top" aria-label="top navigation">
+
+                <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
+                <a href="../index.html">django-freeradius</a>
+
+            </nav>
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

File: docs/build/html/general/freeradius.html [hunk 6]
```diff
@@ -0,0 +1,727 @@
...
+
+
+                </div>
+            </div>
+        </nav>
+
+        <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">
+
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

File: docs/build/html/general/freeradius.html [hunk 7]
```diff
@@ -0,0 +1,727 @@
...
+                        <li class="toctree-l1"><a class="reference internal" href="enforcing_limits.html">Enforcing session limits</a></li>
+                        <li class="toctree-l1"><a class="reference internal" href="registration.html">Registration of new users</a></li>
+                        <li class="toctree-l1"><a class="reference internal" href="registration.html#registration-in-openwisp-radius">Registration in openwisp-radius</a></li>
+                        <li class="toctree-l1"><a class="reference internal" href="social_login.html">Social Login</a></li>
+                        <li class="toctree-l1"><a class="reference internal" href="api.html">API Documentation</a></li>
+                        <li class="toctree-l1"><a class="reference internal" href="abstract_models.html">Abstract Models</a></li>
+                        <li class="toctree-l1"><a class="reference internal" href="contributing.html">Contributing</a></li>
+                        <li class="toctree-l1"><a class="reference internal" href="goals.html">Motivations and Goals</a></li>
+                    </ul>
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

File: docs/build/html/general/freeradius.html [hunk 8]
```diff
@@ -0,0 +1,727 @@
...
+                            <input type="hidden" name="check_keywords" value="yes" />
+                            <input type="hidden" name="area" value="default" />
+                        </form>
+                    </div>
+
+
+                </div>
+
+                <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

File: docs/build/html/general/freeradius.html [hunk 9]
```diff
@@ -0,0 +1,727 @@
...
+          
+          </a>
+
+
+
+
+                    <div class="version">
+                        0.1
+                    </div>
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

File: docs/build/html/general/freeradius.html [hunk 10]
```diff
@@ -0,0 +1,727 @@
...
+                <div class="wy-side-nav-search">
+
+
+
+                    <a href="../index.html" class="icon icon-home"> django-freeradius
+          
+
+          
+          </a>
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

File: docs/build/html/general/freeradius.html [hunk 11]
```diff
@@ -0,0 +1,727 @@
...
+
+        <nav data-toggle="wy-nav-shift" class="wy-nav-side">
+            <div class="wy-side-scroll">
+                <div class="wy-side-nav-search">
+
+
+
+                    <a href="../index.html" class="icon icon-home"> django-freeradius
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

File: docs/build/html/general/freeradius.html [hunk 12]
```diff
@@ -0,0 +1,727 @@
...
+<body class="wy-body-for-nav">
+
+
+    <div class="wy-grid-for-nav">
+
+
+        <nav data-toggle="wy-nav-shift" class="wy-nav-side">
+            <div class="wy-side-scroll">
+                <div class="wy-side-nav-search">
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

File: docs/build/html/general/freeradius.html [hunk 13]
```diff
@@ -0,0 +1,727 @@
...
+    <script src="../_static/js/modernizr.min.js"></script>
+
+</head>
+
+<body class="wy-body-for-nav">
+
+
+    <div class="wy-grid-for-nav">
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

File: docs/build/html/general/freeradius.html [hunk 14]
```diff
@@ -0,0 +1,727 @@
...
+    <link rel="next" title="Management commands" href="management_commands.html" />
+    <link rel="prev" title="Available settings" href="settings.html" />
+
+
+    <script src="../_static/js/modernizr.min.js"></script>
+
+</head>
+
+<body class="wy-body-for-nav">
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

File: docs/build/html/general/freeradius.html [hunk 15]
```diff
@@ -0,0 +1,727 @@
...
+
+
+    <link rel="stylesheet" href="../_static/css/theme.css" type="text/css" />
+    <link rel="stylesheet" href="../_static/pygments.css" type="text/css" />
+    <link rel="index" title="Index" href="../genindex.html" />
+    <link rel="search" title="Search" href="../search.html" />
+    <link rel="next" title="Management commands" href="management_commands.html" />
+    <link rel="prev" title="Available settings" href="settings.html" />
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

File: docs/build/html/general/freeradius.html [hunk 16]
```diff
@@ -0,0 +1,727 @@
...
+<html class="no-js" lang="en">
+<!--<![endif]-->
+
+<head>
+    <meta charset="utf-8">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>Installation and configuration of Freeradius 3 &mdash; django-freeradius 0.1 alpha documentation</title>
+
```
Comment (ppabcd) [inline docs/build/html/general/freeradius.html]: Too many spaces.

**Comment by EdgeKing810** [comment]
I didn't know we (students) could review... :/

**Comment by ppabcd** [comment]
@EdgeKing810 contributor only can review without approve or decline.

**Comment by YashJipkate** [comment]
@ppabcd Actually those spaces were already present there in the repo. Although I have now removed them in my last push.
And I also added newline at end of the file as said by you.

**Comment by ppabcd** [comment]
https://www.freeformatter.com/html-formatter.html maybe will help you and don't forget to add new line at end of file. @YashJipkate

**Comment by YashJipkate** [comment]
@ppabcd used that online formatter of yours and pushed again.

**Comment by marfgold1** [comment]
Don't you want to squash the commits and keep the latest commit message?

**Comment by dwang** [comment]
@YashJipkate 

I see that you're currently editing the built HTML file, but instead you should edit the source RST (reStructuredText) file. You can find it at [docs/source/general/freeradius.rst](https://github.com/openwisp/django-freeradius/blob/master/docs/source/general/freeradius.rst). There should not be any HTML files in the docs at the end. [Here](http://docutils.sourceforge.net/docs/user/rst/quickref.html) is a quick reference to the reStructuredText syntax.

As @marfgold1 just said, once you are done, you should squash your commits using the local `git` client and keep your latest commit message as it is correct. 👍

**Pull request closed without merging**